package.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/ossf/package-feeds/blob/main/package.schema.json",
+    "title": "Package Schema Version 1.0",
+    "description": "The package representation as outputted by a ScheduledFeed",
+    "examples": [
+      "foobarpackage",
+      "1.0.0",
+      "2021-03-30T14:26:28",
+      "npm",
+      "1.0"
+    ],
+    "type": "object",
+    "properties": {
+      "name": {
+        "type": "string",
+        "description": "The name of the package"
+      },
+      "version": {
+        "type": "string",
+        "description": "The package version, formatted respective to the given package type"
+      },
+      "created_date": {
+        "type": "string",
+        "description": "RFC 3339 timestamp representing the package creation date",
+        "format": "date-time"
+      },
+      "type": {
+        "type": "string",
+        "description": "The type of package, this being the `FeedName` of the given package feed"
+      },
+      "schema_ver": {
```

Comment:
instead of `name`, `version` and `type`, I would love to see support for [packageURL](https://github.com/package-url/purl-spec)

Reply:
Sure, with versioning of the schema it should allow for extra metadata(such as packageURL, for example) to be added and optionally consumed by subscribers, whilst not breaking compatibility